infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/server/PreviewUrlExposer.java

```diff
@@ -78,24 +80,31 @@ public void expose(T env) throws InternalInfrastructureException {
               null,
               foundService.get().getMetadata().getName(),
               servicePort,
-              Collections.emptyMap());
+              ImmutableMap.of(
+                  command.getName(),
+                  new ServerConfigImpl(
+                      servicePort.getName(), "http", "/", Collections.emptyMap())));
```

Comment:
what does `http` mean here? Does matter if che is working over https?

Reply:
It IMHO should not matter here, because this describes the service that the preview URL exposes - so the Ingress will be served using https but will forward traffic to the service using http. Or at least that's how I imagine it works :)

Reply:
This is actually the exact thing that I wanted to ask @sparkoo about. I think this is OK given how preview URLs were set up previously, but I am not sure.